src/SFML/Audio/SoundFileWriterWav.cpp

```diff
@@ -43,25 +43,25 @@ namespace
 
 void encode(std::ostream& stream, std::int16_t value)
 {
-    const std::byte bytes[] = {static_cast<std::byte>(value & 0xFF), static_cast<std::byte>(value >> 8)};
-    stream.write(reinterpret_cast<const char*>(bytes), sizeof(bytes));
+    const std::array bytes = {static_cast<char>(value & 0xFF), static_cast<char>(value >> 8)};
+    stream.write(bytes.data(), bytes.size());
```

Comment:
I think I'm the one who started using `std::byte` here. It wasn't the right move since we immediately have to `reintrepret_cast` to a `char*`. Might as well just keeping using `char` to avoid extra casts.